Sources/MapboxNavigation/NavigationViewController.swift

```diff
@@ -387,9 +385,37 @@ open class NavigationViewController: UIViewController, NavigationStatusPresenter
     
     // MARK: - Setting-up methods
     
+    /**
+     Updates key settings before loading the view.
+     
+     This method basically re-runs the setup which takes place in `init`. It could be useful if some of the attribtues have changed before `NavigationViewController` did load it's view, or if you did not have access to initializing logic. For example, as a part of `UIStoryboardSegue` configuration.
```

Comment:
```suggestion
     This method basically re-runs the setup which takes place in `init`. It could be useful if some of the attributes have changed before `NavigationViewController` did load it's view, or if you did not have access to initializing logic. For example, as a part of `UIStoryboardSegue` configuration.
```